.pipelines/e2e-with-billing.yml

```
@@ -48,7 +48,16 @@ stages:
     - script: |
         # Pass variables between tasks: https://medium.com/microsoftazure/how-to-pass-variables-in-azure-pipelines-yaml-tasks-5c81c5d31763
         echo "##vso[task.setvariable variable=REGION]$LOCATION"
-        CLUSTER="v4-e2e-V$BUILD_BUILDID-$LOCATION"
+        # TODO: Remove this hack after AvailabilitySet name too long bug is fixed.
+        NONZONAL_REGIONS="australiacentral australiacentral2 australiasoutheast brazilsoutheast canadaeast centraluseuap eastus2euap japanwest northcentralus norwaywest southafricanorth southindia switzerlandwest uaenorth ukwest westcentralus westus"
+        if echo $NONZONAL_REGIONS | grep -w $LOCATION > /dev/null;
```

Could use `grep -q` for no output, but the same thing really :)
```suggestion
        if echo $NONZONAL_REGIONS | grep -wq $LOCATION
```

---

Thanks for the feedback!  Cleaner solution :)